### PR TITLE
In Production suppress DEBUG, INFO on standard out, and DEBUG on jour…

### PR DESCRIPTION
**Pull request by jlhughes**
…nal logs

### CODE REVIEW

File: src/foam/nanos/logger/StdoutLoggerDAO.js [hunk 1]
```diff
@@ -30,8 +36,16 @@ foam.CLASS({
       LogMessage lm = (LogMessage) getDelegate().put_(x, obj);
       if ( getEnabled() &&
            lm != null ) {
-        System.out.println(lm.getCreated() + ","+lm.getThread()+","+lm.getSeverity()+","+lm.getMessage());
-      } 
+        // Only write WARN, ERROR to SYSLOG in production to reduce
```
Comment (kgrgreer) [inline src/foam/nanos/logger/StdoutLoggerDAO.js:44]: I think your indentation is off by one character.

File: src/foam/nanos/logger/StdoutLoggerDAO.js [hunk 2]
```diff
@@ -30,8 +36,16 @@ foam.CLASS({
...
+        // burden on syslogd, journald. With our own journal logs we are
+        // effectively writing out logs twice. 
+       AppConfig appConfig = (AppConfig) x.get("appConfig");
```
Comment (kgrgreer) [inline src/foam/nanos/logger/StdoutLoggerDAO.js]: Not the end of the world, but couldn't we look up the appConfig just once?

Comment (jlhughes) [reply src/foam/nanos/logger/StdoutLoggerDAO.js]: changed.

File: src/foam/nanos/logger/StdoutLoggerDAO.js (hunk 1 above)
Comment (jlhughes) [reply src/foam/nanos/logger/StdoutLoggerDAO.js:44]: corrected.

**Pull request merged**